
Patches the download HTML file of the CUPS translation guide.

Changes:
Replace <VAR> HTML tag with <I> to fix its italic format convertion to Markdown
Replace Table's <CAPTION> HTML tag with <P><I> to fix its italic format convertion to Markdown, without messing the table's header.
Replace <P CLASS="command"> block with <PRE CLASS="command"> to fix the convertion to Markdown, otherwise its content is not extracted to Markdown

diff --git a/translation/src/translation.html b/translation/src/translation.html
index 7202c4c..b423d95 100644
--- a/translation/src/translation.html
+++ b/translation/src/translation.html
@@ -17,24 +17,24 @@
 
 <UL>
 
-	<LI><VAR>desktop/cups.desktop</VAR> - the GNOME/KDE desktop file pointing to the CUPS web interface</LI>
+	<LI><I>desktop/cups.desktop</I> - the GNOME/KDE desktop file pointing to the CUPS web interface</LI>
 
-	<LI><VAR>doc/index.html.in</VAR> - the web interface home page</LI>
+	<LI><I>doc/index.html.in</I> - the web interface home page</LI>
 
-	<LI><VAR>locale/cups.pot</VAR> - the message catalog</LI>
+	<LI><I>locale/cups.pot</I> - the message catalog</LI>
 
-	<LI><VAR>templates/*.tmpl</VAR> and <VAR>templates/header.tmpl.in</VAR> - the web interface 	template files</LI>
+	<LI><I>templates/*.tmpl</I> and <I>templates/header.tmpl.in</I> - the web interface 	template files</LI>
 
 </UL>
 
-<P>With the exception of the message catalogs and desktop file, localization files are placed in subdirectories under the <VAR>doc</VAR> and <VAR>templates</VAR> using the locale name. Locale names are either <VAR>ll</VAR> or <VAR>ll_CC</VAR>, where "ll" is the 2-letter language code and "CC" is the 2-letter country code. <em>CUPS does not currently use or support the newer <VAR>ll-region</VAR> syntax for locale names.</em></P>
+<P>With the exception of the message catalogs and desktop file, localization files are placed in subdirectories under the <I>doc</I> and <I>templates</I> using the locale name. Locale names are either <I>ll</I> or <I>ll_CC</I>, where "ll" is the 2-letter language code and "CC" is the 2-letter country code. <em>CUPS does not currently use or support the newer <I>ll-region</I> syntax for locale names.</em></P>
 
 <P>All non-image files must be encoded using the UTF-8 character set.</P>
 
 
 <H2 CLASS="title" ID="SUBMIT">Submitting a Translation for CUPS</H2>
 
-<P>To submit a translation for inclusion in CUPS, translate the desktop file, all of the template files, the <VAR>index.html.in</VAR> file, and the message catalog. Place these files in the correct subdirectories in the CUPS source code archive and run the following command to create an archive with your files:</P>
+<P>To submit a translation for inclusion in CUPS, translate the desktop file, all of the template files, the <I>index.html.in</I> file, and the message catalog. Place these files in the correct subdirectories in the CUPS source code archive and run the following command to create an archive with your files:</P>
 
 <PRE CLASS="command">
 tar cvf <I>ll_CC</I>.tar.gz desktop/cups.desktop doc/<I>ll_CC</I> locale/cups_<I>ll_CC</I>.po templates/<I>ll_CC</I>
@@ -47,7 +47,7 @@ tar cvf <I>ll_CC</I>.tar.gz desktop/cups.desktop doc/<I>ll_CC</I> locale/cups_<I
 
 <H2 CLASS="title" ID="DESKTOP">The Desktop File</H2>
 
-<P>The <VAR>desktop/cups.desktop</VAR> file provides a link to the CUPS web interface from desktop environments such as GNOME and KDE. To translate this file, add two lines to the bottom with the <tt>Name</tt> and <tt>Comment</tt> keys:</P>
+<P>The <I>desktop/cups.desktop</I> file provides a link to the CUPS web interface from desktop environments such as GNOME and KDE. To translate this file, add two lines to the bottom with the <tt>Name</tt> and <tt>Comment</tt> keys:</P>
 
 <PRE CLASS="command">
 Name[<I>ll_CC</I>]=<I>Translation of "Manage Printing"</I>
@@ -57,51 +57,51 @@ Comment[<I>ll_CC</I>]=<I>Translation of "CUPS Web Interface"</I>
 
 <H2 CLASS="title" ID="HOMEPAGE">The Home Page</H2>
 
-<P>The <VAR>index.html.in</VAR> file is a complete HTML file that is displayed when the user visits "http://localhost:631/". Edit the existing <VAR>doc/index.html.in</VAR> and save it in the <VAR>doc/ll_CC</VAR> subdirectory so that the configure script can generate it. After configuring, run "make install" in the <VAR>doc</VAR> subdirectory to test the new home page.</P>
+<P>The <I>index.html.in</I> file is a complete HTML file that is displayed when the user visits "http://localhost:631/". Edit the existing <I>doc/index.html.in</I> and save it in the <I>doc/ll_CC</I> subdirectory so that the configure script can generate it. After configuring, run "make install" in the <I>doc</I> subdirectory to test the new home page.</P>
 
 
 <H2 CLASS="title" ID="CATALOGS">Message Catalogs</H2>
 
-<P>CUPS message catalogs are GNU gettext ".po" text files that provide a list of localized message strings for the CUPS software. Message catalogs are named <VAR>cups_ll.po</VAR> or <VAR>cups_ll_CC.po</VAR>, where "ll" is the standard 2-letter abbreviation for the language and "CC" is the standard 2-letter abbreviation for the country.</P>
+<P>CUPS message catalogs are GNU gettext ".po" text files that provide a list of localized message strings for the CUPS software. Message catalogs are named <I>cups_ll.po</I> or <I>cups_ll_CC.po</I>, where "ll" is the standard 2-letter abbreviation for the language and "CC" is the standard 2-letter abbreviation for the country.</P>
 
-<P>When translating a new message catalog, copy the <VAR>cups.pot</VAR> message catalog file in the <VAR>locale</VAR> subdirectory of the CUPS source code. For example, to start translating the message catalog to Canadian French, you would type the following commands:</P>
+<P>When translating a new message catalog, copy the <I>cups.pot</I> message catalog file in the <I>locale</I> subdirectory of the CUPS source code. For example, to start translating the message catalog to Canadian French, you would type the following commands:</P>
 
 <PRE CLASS="command">
 cd locale
 cp cups.pot cups_fr_CA.po
 </PRE>
 
-<P>Alternatively, you can copy the existing <VAR>cups_fr.po</VAR> message catalog and then make any necessary changes.</P>
+<P>Alternatively, you can copy the existing <I>cups_fr.po</I> message catalog and then make any necessary changes.</P>
 
 <P>Once you have make your copy of the file, edit it using your favorite text editor or translation program to translate the text to the desired language.</P>
 
-<P>Then validate your translation using the <VAR>locale/checkpo</VAR> utility:</P>
+<P>Then validate your translation using the <I>locale/checkpo</I> utility:</P>
 
 <PRE CLASS="command">
 cd locale
 ./checkpo cups_ll_CC.po
 </PRE>
 
-<P>After fixing any errors in your translation, add your locale to the <CODE>LANGUAGES</CODE> variable in the <VAR>Makedefs</VAR> file and run the "make install" command in the <VAR>locale</VAR> subdirectory to test the translation. <em>This variable is automatically updated when you run the configure script.</em></P>
+<P>After fixing any errors in your translation, add your locale to the <CODE>LANGUAGES</CODE> variable in the <I>Makedefs</I> file and run the "make install" command in the <I>locale</I> subdirectory to test the translation. <em>This variable is automatically updated when you run the configure script.</em></P>
 
 
 <H2 CLASS="title" ID="TEMPLATES">Template Files</H2>
 
 <P>The CUPS scheduler provides a web interface that can be used to do many common printing and administration tasks. The built-in web server supports localization of web pages through the use of subdirectories for each locale, e.g. "fr" for French, "de" for German, "fr_ca" for French in Canada, and so forth.</P>
 
-<P>Template files are HTML files with special formatting characters in them that allow substitution of variables and arrays. The CUPS CGI programs (<CODE>admin.cgi</CODE>, <CODE>classes.cgi</CODE>, <CODE>help.cgi</CODE>, <CODE>jobs.cgi</CODE>, and <CODE>printers.cgi</CODE>) use these template file to provide dynamic content for the web interface. Template files are installed in the <VAR>/usr/share/cups/templates</VAR> directory by default. <A HREF="table1">Table 1</a> lists the various template files and their purpose.</P>
+<P>Template files are HTML files with special formatting characters in them that allow substitution of variables and arrays. The CUPS CGI programs (<CODE>admin.cgi</CODE>, <CODE>classes.cgi</CODE>, <CODE>help.cgi</CODE>, <CODE>jobs.cgi</CODE>, and <CODE>printers.cgi</CODE>) use these template file to provide dynamic content for the web interface. Template files are installed in the <I>/usr/share/cups/templates</I> directory by default. <A HREF="table1">Table 1</a> lists the various template files and their purpose.</P>
 
 <P>Translated versions of the template files should be saved in
-the <VAR>templates/ll_CC</VAR> subdirectory. For example,
+the <I>templates/ll_CC</I> subdirectory. For example,
 Canadian French template files should be saved in the
-<VAR>templates/fr_CA</VAR> subdirectory. After you have
+<I>templates/fr_CA</I> subdirectory. After you have
 translated all of the templates, add the locale to the
 <CODE>LANGUAGES</CODE> variable in the
-<VAR>Makedefs</VAR> file and run "make install" in the
-<VAR>templates</VAR> subdirectory to test the translation.</P>
+<I>Makedefs</I> file and run "make install" in the
+<I>templates</I> subdirectory to test the translation.</P>
 
 <DIV CLASS="table"><TABLE ID="table1" SUMMARY="Web Interface Template Files">
-<CAPTION>Table 1: Web Interface Template Files</CAPTION>
+<P><I>Table 1: Web Interface Template Files</I></P>
 <THEAD>
 <TR>
 	<TH>Filename</TH>
@@ -191,7 +191,7 @@ translated all of the templates, add the locale to the
 </TR>
 <TR>
 	<TD>header.tmpl(.in)</TD>
-	<TD>This template is used as the standard header on all dynamic content. Edit the <VAR>header.tmpl.in</VAR> file and let the configure script generate the <VAR>header.tmpl</VAR> file.</TD>
+	<TD>This template is used as the standard header on all dynamic content. Edit the <I>header.tmpl.in</I> file and let the configure script generate the <I>header.tmpl</I> file.</TD>
 </TR>
 <TR>
 	<TD>help-header.tmpl</TD>
@@ -420,33 +420,33 @@ translated all of the templates, add the locale to the
 {<I>variable</I>><I>value</I>?<I>true</I>:<I>false</I>}
 </PRE>
 
-<P>where <VAR>true</VAR> is the text that is included if the condition is true and <VAR>false</VAR> is the text that is included if the condition is false. A value of <CODE>#</CODE> is replaced with the current element number (starting at 1.) The character after the variable name specifies the condition to test. <A HREF="#table2">Table 2</A> shows the available test conditions.</P>
+<P>where <I>true</I> is the text that is included if the condition is true and <I>false</I> is the text that is included if the condition is false. A value of <CODE>#</CODE> is replaced with the current element number (starting at 1.) The character after the variable name specifies the condition to test. <A HREF="#table2">Table 2</A> shows the available test conditions.</P>
 
 <DIV CLASS="table"><TABLE ID="table2" SUMMARY="Template Substitution Conditions">
-<CAPTION>Table 2: Template Substitution Conditions</CAPTION>
+<P><I>Table 2: Template Substitution Conditions</I></P>
 <TR>
 	<TH>Char</TH>
 	<TH>Condition</TH>
 </TR>
 <TR>
 	<TD>?</TD>
-	<TD>True if <VAR>variable</VAR> exists.</TD>
+	<TD>True if <I>variable</I> exists.</TD>
 </TR>
 <TR>
 	<TD>=</TD>
-	<TD>True if <VAR>variable</VAR> is equal to <VAR>value</VAR>.</TD>
+	<TD>True if <I>variable</I> is equal to <I>value</I>.</TD>
 </TR>
 <TR>
 	<TD>!</TD>
-	<TD>True if <VAR>variable</VAR> is not equal to <VAR>value</VAR>.</TD>
+	<TD>True if <I>variable</I> is not equal to <I>value</I>.</TD>
 </TR>
 <TR>
 	<TD>&lt;</TD>
-	<TD>True if <VAR>variable</VAR> is less than <VAR>value</VAR>.</TD>
+	<TD>True if <I>variable</I> is less than <I>value</I>.</TD>
 </TR>
 <TR>
 	<TD>></TD>
-	<TD>True if <VAR>variable</VAR> is greater than <VAR>value</VAR>.</TD>
+	<TD>True if <I>variable</I> is greater than <I>value</I>.</TD>
 </TR>
 </TABLE></DIV>
 
@@ -454,20 +454,19 @@ translated all of the templates, add the locale to the
 
 <P>Because HTML contains both markup (that generally is not localized) and text (which <em>is</em> localized), you should carefully avoid changing the markup unless it contains human-readable text. The following HTML examples show how to correctly localize template files:</P>
 
-<P CLASS="command">
-&lt;!-- English table heading --&gt;<br>
-&lt;table class="list" summary="Printer List"><br>
-<br>
-&lt;!-- Correctly localized to French; the class attribute is unchanged but summary is localized --><br>
-&lt;table class="list" summary="Liste des imprimantes"><br>
-<br>
-<br>
-&lt;!-- English hyperlink --><br>
-&lt;li>&lt;a {SECTION=help?class="active" :}href="/help/">Help&lt;/a>&lt;/li><br>
-<br>
-&lt;!-- Correctly localized to Danish; the href attribute is unchanged while the link text is localized --><br>
-&lt;li>&lt;a {SECTION=help?class="active" :}href="/help/">Hjælp&lt;/a>&lt;/li><br>
-</P>
+<PRE CLASS="command">
+&lt;!-- English table heading --&gt;
+&lt;table class="list" summary="Printer List">
+
+&lt;!-- Correctly localized to French; the class attribute is unchanged but summary is localized -->
+&lt;table class="list" summary="Liste des imprimantes">
+
+&lt;!-- English hyperlink -->
+&lt;li>&lt;a {SECTION=help?class="active" :}href="/help/">Help&lt;/a>&lt;/li>
+
+&lt;!-- Correctly localized to Danish; the href attribute is unchanged while the link text is localized -->
+&lt;li>&lt;a {SECTION=help?class="active" :}href="/help/">Hjælp&lt;/a>&lt;/li>
+</PRE>
 
 
 <H3 ID="CGI">CGI Programs</H3>
@@ -489,7 +488,7 @@ translated all of the templates, add the locale to the
 <P>In addition, the <CODE>classes.cgi</CODE>, <CODE>jobs.cgi</CODE>, and <CODE>printers.cgi</CODE> programs support a <CODE>WHICH_JOBS</CODE> variable to control which jobs are displayed. <A HREF="#table3">Table 3</A> lists the supported values.</P>
 
 <DIV CLASS="table"><TABLE ID="table3" SUMMARY="WHICH_JOBS Values">
-<CAPTION>Table 3: WHICH_JOBS Values</CAPTION>
+<P><I>Table 3: WHICH_JOBS Values</I></P>
 <THEAD>
 <TR>
 	<TH>WHICH_JOBS Value</TH>
@@ -512,10 +511,10 @@ translated all of the templates, add the locale to the
 
 <H4 ID="admin.cgi">admin.cgi</H4>
 
-<P>The <CODE>admin.cgi</CODE> program handles all of the printer and class administration functions and is run for all direct accesses to the <VAR>/admin</VAR> resource. For most operations it uses the <CODE>PRINTER_NAME</CODE> and <CODE>OP</CODE> form variables to specify the action requested. <A HREF="#table4">Table 4</A> shows the supported <CODE>OP</CODE> values.</P>
+<P>The <CODE>admin.cgi</CODE> program handles all of the printer and class administration functions and is run for all direct accesses to the <I>/admin</I> resource. For most operations it uses the <CODE>PRINTER_NAME</CODE> and <CODE>OP</CODE> form variables to specify the action requested. <A HREF="#table4">Table 4</A> shows the supported <CODE>OP</CODE> values.</P>
 
 <DIV CLASS="table"><TABLE ID="table4" SUMMARY="admin.cgi OP Values">
-<CAPTION>Table 4: admin.cgi OP Values</CAPTION>
+<P><I>Table 4: admin.cgi OP Values</I></P>
 <THEAD>
 <TR>
 	<TH>OP Value</TH>
@@ -583,10 +582,10 @@ translated all of the templates, add the locale to the
 
 <H4 ID="classes.cgi">classes.cgi</H4>
 
-<P>The <CODE>classes.cgi</CODE> program is responsible for listing class information, including jobs destined for that class. It is for all direct accesses to the <VAR>/classes</VAR> resource and supports the optional form variables <CODE>OP</CODE> and <CODE>WHICH_JOBS</CODE>. If no form variables are supplied then the CGI lists all or a specific class and the active jobs on each class. <A HREF="#table5">Table 5</A> shows the supported <CODE>OP</CODE> values.</P>
+<P>The <CODE>classes.cgi</CODE> program is responsible for listing class information, including jobs destined for that class. It is for all direct accesses to the <I>/classes</I> resource and supports the optional form variables <CODE>OP</CODE> and <CODE>WHICH_JOBS</CODE>. If no form variables are supplied then the CGI lists all or a specific class and the active jobs on each class. <A HREF="#table5">Table 5</A> shows the supported <CODE>OP</CODE> values.</P>
 
 <DIV CLASS="table"><TABLE ID="table5" SUMMARY="classes.cgi OP Values">
-<CAPTION>Table 5: classes.cgi OP Values</CAPTION>
+<P><I>Table 5: classes.cgi OP Values</I></P>
 <THEAD>
 <TR>
 	<TH>OP Value</TH>
@@ -628,15 +627,15 @@ translated all of the templates, add the locale to the
 
 <H4 ID="help.cgi">help.cgi</H4>
 
-<P>The <CODE>help.cgi</CODE> program handles all of the on-line help functions and is run for all direct accesses to the <VAR>/help</VAR> resource.</P>
+<P>The <CODE>help.cgi</CODE> program handles all of the on-line help functions and is run for all direct accesses to the <I>/help</I> resource.</P>
 
 
 <H4 ID="jobs.cgi">jobs.cgi</H4>
 
-<P>The <CODE>jobs.cgi</CODE> program handles all of the job functions and is run for all direct accesses to the <VAR>/jobs</VAR> resource. For most operations it uses the <CODE>JOB_ID</CODE>, <CODE>OP</CODE>, and <CODE>WHICH_JOBS</CODE> form variables to specify the action requested. <A HREF="#table6">Table 6</A> shows the supported <CODE>OP</CODE> values.</P>
+<P>The <CODE>jobs.cgi</CODE> program handles all of the job functions and is run for all direct accesses to the <I>/jobs</I> resource. For most operations it uses the <CODE>JOB_ID</CODE>, <CODE>OP</CODE>, and <CODE>WHICH_JOBS</CODE> form variables to specify the action requested. <A HREF="#table6">Table 6</A> shows the supported <CODE>OP</CODE> values.</P>
 
 <DIV CLASS="table"><TABLE ID="table6" SUMMARY="jobs.cgi OP Values">
-<CAPTION>Table 6: jobs.cgi OP Values</CAPTION>
+<P><I>Table 6: jobs.cgi OP Values</I></P>
 <THEAD>
 <TR>
 	<TH>OP Value</TH>
@@ -671,10 +670,10 @@ translated all of the templates, add the locale to the
 
 <H4 ID="printers.cgi">printers.cgi</H4>
 
-<P>The <CODE>printers.cgi</CODE> program is responsible for listing printer information, including jobs destined for that printer. It is for all direct accesses to the <VAR>/printers</VAR> resource and supports the optional form variables <CODE>OP</CODE> and <CODE>WHICH_JOBS</CODE>. If no form variables are supplied then the CGI lists all printers or a specific printer and the active jobs on that printer. <A HREF="#table7">Table 7</A> shows the supported <CODE>OP</CODE> values.</P>
+<P>The <CODE>printers.cgi</CODE> program is responsible for listing printer information, including jobs destined for that printer. It is for all direct accesses to the <I>/printers</I> resource and supports the optional form variables <CODE>OP</CODE> and <CODE>WHICH_JOBS</CODE>. If no form variables are supplied then the CGI lists all printers or a specific printer and the active jobs on that printer. <A HREF="#table7">Table 7</A> shows the supported <CODE>OP</CODE> values.</P>
 
 <DIV CLASS="table"><TABLE ID="table7" SUMMARY="printers.cgi OP Values">
-<CAPTION>Table 7: printers.cgi OP Values</CAPTION>
+<P><I>Table 7: printers.cgi OP Values</I></P>
 <THEAD>
 <TR>
 	<TH>OP Value</TH>
